django_project/ta_system/views.py

```diff
@@ -1,6 +1,14 @@
 from django.shortcuts import render, redirect
+from .models import SystemStatus
 from django.contrib.auth.decorators import login_required
 
+
 @login_required
 def home(request):
-    return render(request, 'ta_system/home.html')
+    date_list = SystemStatus.objects.order_by('id')
+    date_list = date_list.reverse()
```

Comment:
You can chain this .reverse() call on the end of the .order_by('id') call if you'd like, though it's fine as is too.

Reply:
simplified the code to one line